## <center>Практическое задание к лекции 6.

**1.Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.**

Поскольку нам известно среднее квадратическое отклонение генеральной совокупности $\sigma=16$ , то для расчета $95\%$-го доверительного интервала, будем использовать $Z$-критерий.

Находим значение $Z$ для интервала $0.95$

In [2]:
import numpy as np
import math
from scipy.stats import norm
from scipy.stats import t

In [6]:
z=norm.ppf((1+0.95)/2)
print (f'значение Z для интревала 0,95 составляет {z: .2f}')

значение Z для интревала 0,95 составляет  1.96


Далее находим интервал

In [11]:
d=z*16/math.sqrt(256)
left=80-d
right=80+d
print(f'Доверительный интервал для оценки мат. ожидания генеральной совокупности: [{left: .2f};{right: .2f}].')

Доверительный интервал для оценки мат. ожидания генеральной совокупности: [ 78.04; 81.96].


--- 
### Ответ:
Доверительный интервал для оценки мат. ожидания генеральной совокупности: $\left[78.04;81.96\right]$

**2.В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1. Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.**

In [24]:
arr=np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
n=arr.size
std=arr.std(ddof=1)
mean=arr.mean()
z=norm.ppf((1+0.95)/2)
d=z*std/math.sqrt(n)
left= mean-d
right= mean+d
print(f'95%-й доверительный интервал для истинного значения Х:[{left: .2f};{right: .2f}].')


95%-й доверительный интервал для истинного значения Х:[ 6.31; 6.87].


--- 
### Ответ:
$95\%$-й доверительный интервал для истинного значения $Х$:$\left[6.268;6.912\right]$

**3.Рост дочерей 175, 167, 154, 174, 178, 148, 160, 167, 169, 170**             
**Рост матерей 178, 165, 165, 173, 168, 155, 160, 164, 178, 175**               
**Используя эти данные построить 95% доверительный интервал для разности среднего роста родителей и детей.**

In [30]:
import numpy
daughters = numpy.array([175, 167, 154, 174, 178, 148, 160, 167, 169, 170])
mothers = numpy.array([178, 165, 165, 173, 168, 155, 160, 164, 178, 175])

mean = numpy.mean(mothers) - numpy.mean(daughters)
std = numpy.sqrt((numpy.std(mothers) ** 2 / len(mothers)) + (numpy.std(daughters) ** 2 / len(daughters)))
a = 1.96 * std
left=mean - a
right=mean + a

print(f"95% доверительный интервал разности среднего роста родителей и детей:[{left: .2f};{right: .2f}].", )

95% доверительный интервал разности среднего роста родителей и детей:[-5.33; 9.13].


--- 
### Ответ:
$95\%$-й доверительный интервал разности среднего роста родителей и детей:$\left[-5.33;9.13\right]$